charts/move2kube/values.yaml

```diff
@@ -0,0 +1,16 @@
+sshSecretName: sshkeys
```

Comment:
description will be nice :-) 
you can run `helm lint` to check for warnings

Reply:
will do both :)

Reply:
Be aware that @rgolangh is submitting a GH action to lint the `orchestrator` chart. You probably can reuse it to extend the linting to this new chart

Reply:
added a comment to describe
and lint is ok:
```
$ helm lint
==> Linting .

1 chart(s) linted, 0 chart(s) failed
```